h2o-algos/src/main/java/hex/tree/SharedTree.java

```diff
@@ -165,6 +167,26 @@ protected void ignoreInvalidColumns(int npredictors, boolean expensive) {
       _ncols = _train.numCols()-(isSupervised()?1:0)-numSpecialCols();
 
     PlattScalingHelper.initCalibration(this, _parms, expensive);
+
+    if (expensive) {
```

Comment:
looks like duplicated code, should be moved and reused

Reply:
fixed